app/assets/stylesheets/dark-mode.css

```diff
@@ -135,3 +135,6 @@ progress-circle__inner {
 .progress-circle__content {
   color: var(--dark-bg-accent) !important;
 }
+.advert__icon {
```

Comment:
Nit: can we include a new line between this rule and the one above, please?

Reply:
Yes, you're right, and now is fixed 👍🏽 .